admin/controllers/school.js

```diff
@@ -32,17 +32,17 @@ const getHome = async (req, res, next) => {
 
 const getPupils = async (req, res, next) => {
   res.locals.pageTitle = 'Pupil register'
-  const { sortColumn, sortOrder } = req.params
+  const {sortColumn, sortOrder} = req.params
   res.locals.sortColumn = sortColumn || 'lastName'
   const order = JSON.parse(sortOrder)
   res.locals.sortOrder = typeof order === 'boolean' ? !order : true
   res.locals.sortClass = order === false ? 'triangle down' : 'triangle'
-  const { pupils } = await fetchPupilsData(req.user.School)
+  const {pupils} = await fetchPupilsData(req.user.School)
   let pupilsFormatted = await Promise.all(pupils.map(async (p) => {
-    const { foreName, lastName, _id } = p
+    const {foreName, lastName, _id} = p
     const dob = moment(p.dob).format('DD/MM/YYYY')
     const answers = await fetchPupilAnswers(p._id)
-    const { score } = fetchScoreDetails(answers)
```

Comment:
codacy doesn't like spaces on destructured variables?
I prefer it this way honestly, sounds very subjective.

Reply:
I agree.  I have reconfigured my editor whitespace settings:

![screen shot 2017-09-19 at 08 52 03](https://user-images.githubusercontent.com/13945069/30581423-faf8828c-9d17-11e7-9e65-aeb7d14a720d.png)